octoprint_mrbeam/static/js/app/view-models/working-area.js

```diff
@@ -626,7 +626,7 @@ $(function () {
                     duration_processing: Date.now() - start_ts,
                     gco_generator_info: {
                         generator: beamos_vers ? "beamOS" : null,
-                        version: beamos_vers ? beamos_vers : null,
+                        version: beamos_vers,
```

Comment:
why you removed this?

Reply:
it is meaningless as beamos_vers can either be null or not so the ternary conditional operator is not needed